### PR TITLE
Feature/3 ci cd

### PR DESCRIPTION
**Pull request by John3107**
create file with linting
set up autolinting
finish ci/cd task

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    },
+    "plugins": ["@typescript-eslint", "prettier", "react", "jest", "effector"],
+    "rules": {
+      "no-prototype-builtins": "off",
+      "eqeqeq": "error",
+      "react/prop-types": "off",
+      "radix": "warn",
+      "no-console": ["warn", { "allow": ["debug", "warn", "error", "assert"] }]
+    }
+  }
```
Comment (vladdoletskiy) [inline .eslintrc.json:33]: add 1 line after code

File: src/App.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.app{
```
Comment (vladdoletskiy) [inline src/App.scss:1]: ?

**Review by vladdoletskiy — APPROVED** [review]

File: .eslintrc.json (hunk 1 above)
Comment (John3107) [reply .eslintrc.json:33]: I will add it in new pull request

File: src/App.scss (hunk 2 above)
Comment (John3107) [reply src/App.scss:1]: It will be corrected in new pull request

**Pull request merged**